scripts/zones/La_Theine_Plateau/npcs/Vicorpasse.lua

```diff
@@ -27,7 +27,7 @@ function onTrigger(player,npc)
 	if(player:getCurrentMission(SANDORIA) == THE_RESCUE_DRILL) then
 		local MissionStatus = player:getVar("MissionStatus");
 		
-		if(MissionStatus == 4) then
+		if(MissionStatus >= 0 and MissionStatus <= 4) then

```

Comment:
it's ok, i know NA wiki is in some way different from JPs. lol
our wiki says we can directly talk to Vicorpasse:
http://wiki.ffo.jp/html/8247.html
http://elemen.jp/ff11/mission/san/index.html#RANK2_1